Assets/MixedRealityToolkit.SDK/Features/Utilities/InputUtils.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.WindowsDevicePortal;
+using System.Runtime.InteropServices.WindowsRuntime;
+using UnityEngine;
+using UnityEngine.Experimental.XR;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Utilities for accessing input state.
+    /// </summary>
+    public static class InputUtils
```

Comment:
nit: Maybe use something less generic. InputUtils is a big catch-all bucket. InputStateutils or InputRayUtils or etc

Reply:
InputRayUtils sounds good to me, thanks troy!